test/integration/exit.js

```diff
@@ -0,0 +1,54 @@
+const chai = require('chai');
```

Comment:
please modify the integration test for `autolabjs init` command. As can be seen from [travis log](), there is a leakage of test-specific information to console. This leak must be prevented.

Reply:
This is the output of the *inquirer*, which is not using ``console.log``. I need to look at the source code and see how it is managing the output.

Reply:
Can we mock inquirer itself?

Reply:
With sinon, we cannot do it. This is because sinon is entirely function-based.